### PR TITLE
New term entry for PHP about the strpos() function

### PR DESCRIPTION
**Pull request by Judis07**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by caupolicandiaz** [comment]
@Judis07 This entry was recently completed by another contributor. Please look for another issue to work on. Thanks
https://github.com/Codecademy/docs/pull/2618

**Comment by Judis07** [comment]
Hi, please correct if I am wrong. I think #2618 is for another method called stripos() while this issue is for strpos()? While both method are very much similar but one is case sensitive and the other one case insensitive. Thanks

**Comment by caupolicandiaz** [comment]
> Hi, please correct if I am wrong. I think #2618 is for another method called stripos() while this issue is for strpos()? While both method are very much similar but one is case sensitive and the other one case insensitive. Thanks

My mistake.

File: content/php/concepts/string-functions/terms/strops/strops.md
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/strops/strops.md:11]:
```suggestion
```

Comment (caupolicandiaz) [reply content/php/concepts/string-functions/terms/strops/strops.md:11]: only two items are permitted under subjects

File: content/php/concepts/string-functions/terms/strops/strops.md
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/strops/strops.md:1]: Please review our standards and update the metadata, and spacing formatting.

File: content/php/concepts/string-functions/terms/strops/strops.md
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/strops/strops.md:4]:
```suggestion
Description: 'Performs a case-sensitive search for the first occurrence of a substring in a given string.'
```

File: content/php/concepts/string-functions/terms/strops/strops.md
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/strops/strops.md:60]: This is essentially the same example. Codebytes should not duplicate other examples within the entry. Please revise the example or delete it.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Judis07 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by Judis07** [comment]
Hello @caupolicandiaz, I have made the changes you suggested. Please review them. 
Thanks

**Comment by Judis07** [comment]
@caupolicandiaz please let me know if you need any new changes from my end

**Review by caupolicandiaz — APPROVED** [review]
@Judis07 Overall, this looks good. There were changes to the metadata that were still missing, which I added (please review for future reference). I also made some edits to the example. This is ready for a second review.

**Comment by Judis07** [comment]
@caupolicandiaz Do let me know if anything needs to be done from my end. Thanks

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz Do let me know if anything needs to be done from my end. Thanks

The next step is for another maintainer to review this entry. If they have any suggestions you will need to make additional edits to complete this submission.

**Review by SSwiniarski — APPROVED** [review]
@Judis07 Looks good.

**Comment by SSwiniarski** [comment]
@Judis07 Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/string-functions/strops

image

**Comment by SSwiniarski** [comment]
@Judis07 I've noticed there is a typo in the filename. Can you submit a new PR that changes the path from `content/php/concepts/string-functions/terms/strops/strops.md` to `content/php/concepts/string-functions/terms/strpos/strpos.md`

**Comment by Judis07** [comment]
@SSwiniarski sure, I will submit a new PR with this fix.

**Pull request merged**
